.vscode/settings_template.json

```diff
@@ -1,11 +1,11 @@
 {
     "r.alwaysUseActiveTerminal": true,
     "r.bracketedPaste": true,
-    "r.rpath.mac": "/opt/homebrew/bin/R",
+    "r.rpath.mac": "Insert output of `which R` (MacOS, Linux, WSL) or `where.exe R` (Windows Powershell)",
```

Comment:
I'm guessing the option used is probably different if you aren't on MacOS (unless I'm misunderstanding  what `path.mac` means)

Reply:
Excellent point!